### DISCUSSION TITLE
Problema pagelle

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#141] [Richieste di aiuto] [answered]
Buongiorno ,quando stampo le pagelle non so il perché , ma come luogo mi esce sempre CAGLIARI , come posso fare?

**Comment by trinko** [comment]
Hai aggiornato all'ultima versione?
Non dovrebbe succedere, se hai inserito i dati dell'Istituto e della sede.

**Reply by sempliciniluigi89** [reply]
Si,il registro è stato aggiornato alla versione attuale, ma in tutti i documenti riporta CAGLIARI come luogo per non parlare di quando si va a stampare il programma che riporta sulla testata word la via della scuola CAGLIARI

**Reply by manuelbarberi** [reply]
Cancellato la cache ?

**Comment by sempliciniluigi89** [comment]
Sì

Il Lun 31 Gen 2022, 10:46 Manu ***@***.***> ha scritto:

**Comment by trinko** [comment, marked as answer]
Ho aggiunto di recente delle modifiche alla procedura dello scrutinio, per cui alcuni di questi problemi dovrebbero essere risolti.
Non tutti, però: esistono ancora documenti per i quali è necessario modificare i template o direttamente il codice PHP che gestisce lo scrutinio.

Purtroppo queste modifiche richedono tempo e molta attenzione per non introdurre errori.
